test/unit/base62.test.js

```diff
@@ -0,0 +1,17 @@
+import base62 from 'next/dist/build/webpack/config/blocks/css/loaders/utils/base62'
+
+describe('base62 tests', () => {
+  it('returns first word in dictionary if input equals 0', () => {
+    const result = base62(0)
+
+    expect(result).toBe('0')
+  })
+
+  it('correctly encodes value', () => {
```

Comment:
I'd add a test for your rollover indices as well, just to validate they're working independently from the `123456` case.

Reply:
I don't know what do you mean by rollover indices in base62, could you elaborate a bit, please?